components/ExpandableText.js

```diff
@@ -1,62 +1,107 @@
-import React from 'react';
+import React, { useState, useEffect, useRef } from 'react';
+import { t } from 'ttag';
 import { truncate } from 'lib/text';
+import { withStyles, makeStyles } from '@material-ui/core';
 
-export default class ExpandableText extends React.Component {
-  static defaultProps = {
-    children: '',
-    wordCount: 140,
-  };
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100%',
+    position: 'relative',
+    overflow: 'hidden',
+  },
+  toggleButtonContainer: {
+    position: 'absolute',
+    right: 0,
+    bottom: 0,
+    paddingLeft: '3em',
+    backgroundImage: `linear-gradient(90deg, rgba(255,255,255,0) 0%, ${theme.palette.common.white} 30%, ${theme.palette.common.white} 100%)`,
+  },
+}));
 
-  state = {
-    isExpanded: false,
-  };
+const ToggleButton = withStyles({
+  root: {
+    border: 0,
+    color: '#2079F0',
+    outline: 'none',
+    cursor: 'pointer',
+    background: 'transparent',
+  },
+})(({ classes, toggleExpand, expanded }) => (
+  <button className={classes.root} onClick={toggleExpand}>
+    ({expanded ? t`Show Less ▲` : t`Show More ▼`})
```

Comment:
We can use this so that ▲ ▼ translators don't need to handle the triangles:
```javascript
({expanded ? t`Show Less` + ' ▲' : t`Show More` + ' ▼'})
```

or

```javascript
({expanded ? `${t`Show Less`} ▲` : `${t`Show More`}  ▼`})
```